docs/Tutorial_VALMOD.ipynb

```diff
@@ -0,0 +1,688 @@
+{
```

Comment:
<p>IIUC, Keogh often refers to this as the "twin freak" problem where the top-1 neighbor is similar but the neighbors after that are quite far away.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Thanks!!! I knew I read something about such case before but I couldn't remember the name. I will use the same terminology. I will try to provide the reference if I can find that link/file. I will revise it in the next set of commits.